client/my-sites/importer/author-mapping-pane.jsx

```diff
@@ -58,7 +58,7 @@ class AuthorMappingPane extends React.PureComponent {
 	getMappingDescription = ( numSourceUsers, numTargetUsers, targetTitle, sourceType ) => {
 		if ( numTargetUsers === 1 && numSourceUsers === 1 ) {
 			return this.props.translate(
-				'We found one author on your %(sourceType)s site. ' +
+				'There is one author on your %(sourceType)s site. ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'We found one author on your %(sourceType)s site. Because you're the only author on {{b}}%(destinationSiteTitle)s{{/b}}, all imported content will be assigned to you. Click Start import to proceed.' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/importer-styles)